client/app/components/composites/Branding/Branding.js

```diff
@@ -0,0 +1,26 @@
+import { PropTypes } from 'react';
+import { div, p, a } from 'r-dom';
+
+import { t } from '../../../utils/i18n';
+
+import css from './Branding.css';
+
+export default function Branding({ linkToSharetribe }) {
+  const learnMoreLink = a({ href: linkToSharetribe }, t('web.branding.learn_more'));
+  const sharetribeLink = a({ href: linkToSharetribe }, 'Sharetribe');
+
+  return div({
+    className: css.brandingContainer,
```

Comment:
By not giving classname handles, it makes harder for endusers to style or hide this stuff - which is probably OK?